app/src/main/java/be/scri/views/MyKeyboardView.kt

```diff
@@ -967,29 +967,18 @@ class MyKeyboardView
             )
         }
 
-        private fun openPopupIfRequired(me: MotionEvent): Boolean {
-            // Check if we have a popup layout specified first.
-            if (mPopupLayout == 0) {
-                return false
-            }
-
-            if (mCurrentKey < 0 || mCurrentKey >= mKeys.size) {
-                return false
-            }
-
-            val popupKey = mKeys[mCurrentKey]
-            val result = onLongPress(popupKey, me)
-            if (result) {
-                mAbortKey = true
-                showPreview(NOT_A_KEY)
-            }
+    private fun openPopupIfRequired(me: MotionEvent): Boolean {
+        if (mPopupLayout == 0 || mCurrentKey !in mKeys.indices) {
+            return false
+        }
 
-            return result
+        val popupKey = mKeys[mCurrentKey]
+        val result = onLongPress(popupKey, me)
+        if (result) {
+            mAbortKey = true
+            showPreview(NOT_A_KEY)
         }
 
-        /**
```

Comment:
Could the comment be opened and closed properly?

Reply:
I'll fix this and the above and we'll see what the state of it all is :)